packages/contracts-bedrock/test/L1/OptimismPortal.t.sol

```diff
@@ -156,6 +156,102 @@ contract OptimismPortal_Test is CommonTest {
         assertEq(address(optimismPortal).balance, _value);
     }
 
+    /// @dev Tests that `depositTransaction` succeeds when msg.sender == tx.origin and non-custom gas is used.
+    function testFuzz_depositTransaction_senderIsOrigin_succeeds(
+        address _to,
+        uint256 _mint,
+        uint256 _value,
+        uint64 _gasLimit,
+        bool _isCreation,
+        bytes memory _data
+    )
+        external
+    {
+        if (_isCreation) {
+            vm.assume(_to == address(0));
```

Comment:
This is going to cause a lot of dropped fuzz runs, perhaps we just say if `_isCreation` then `_to = address(0)`